### ISSUE TITLE
pycrypto is unmaintained and makes it impossible to install gitlab-tools on Debian Bullseye

### ISSUE THREAD

**Issue by Findus23** [Salamek/gitlab-tools#31]
```
> sudo apt depends python3-gitlab-tools 
python3-gitlab-tools
  Depends: python3-celery
  Depends: python3-cron-descriptor
  Depends: python3-crypto (>= 2.6.1)
[...]
```

`python3-crypto` was removed from Debian in the upcoming Bullseye release: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=972184

`python3-cryptography` seems to be a successor, but for whatever reason even after installing `python3-pycryptodome` `import Crypto` still doesn't work.

**Comment by Findus23** [comment]
It turns out the way `python3-pycryptodome` is packaged on Debian, one needs to replace all `import Crypto` with `import Cryptodome`

**Comment by Salamek** [comment]
@Findus23 so python3-pycrodome is not https://pypi.org/project/pycryptodome but https://pypi.org/project/pycryptodomex/ ?

**Comment by Findus23** [comment]
@Salamek Yes, exactly

**Comment by Salamek** [comment]
This is solved in version 1.2.3
